bin/lint-i18n-translations.js

```diff
@@ -0,0 +1,55 @@
+/*
+    Validating i18n translation keys
+    Translation files should match following rules:
+    1. All supported languages should have the same set of keys
+    2. Each key should be on the same line in JSON-file for each language
+*/
+const fs = require('fs')
+const { diffLinesUnified, diffLinesRaw } = require('jest-diff')
+
+const langDir = 'apps/condo/lang'
```

Comment:
it's more reusable if you can detect this path or discover it by working directory. It's help to use this file for others applications

Reply:
You can use something similar to `lint-schema.js`